mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/setting/SettingScreenTest.kt

```diff
@@ -26,8 +26,9 @@ class SettingScreenTest {
     override val pastGamesCount = 10
     override val puzzlesCount = 12
 
-    override fun onEditClick() = Unit
     override fun onSettingsClick() = Unit
+    override fun openEditProfileName() = Unit
```

Comment:
You should stick to our existing convention of naming the callbacks `onBlahBlah`. `onEditProfileNameClick` may be an appropriate name for example.

Reply:
Ah there is a convention. Good to know -> is changed